chain/client/src/test_utils/mod.rs

```diff
@@ -0,0 +1,17 @@
+use near_epoch_manager::RngSeed;
+
+pub mod block_stats;
+pub mod client;
+pub mod peer_manager_mock;
+pub mod setup;
+pub mod test_env;
+pub mod test_env_builder;
+
+pub use block_stats::*;
+pub use client::*;
+pub use peer_manager_mock::*;
+pub use setup::*;
+pub use test_env::*;
+pub use test_env_builder::*;
```

Comment:
It feels like we can use only one of `pub mod` or `pub use`. But I can imagine that it can lead to more chore work, so no strong opinion here.

Reply:
I had added in the use so that the current references to these functions doesn't change, otherwise we would need to write them as `use near_client::test_utils::test_env_builder::TestEnvBuilder;` instead of `use near_client::test_utils::TestEnvBuilder;`